be/src/util/mem_info.cpp

```diff
@@ -35,36 +35,58 @@ bool MemInfo::_s_initialized = false;
 int64_t MemInfo::_s_physical_mem = -1;
 
 void MemInfo::init() {
-    // Read from /proc/meminfo
-    std::ifstream meminfo("/proc/meminfo", std::ios::in);
-    std::string line;
+    // check if application is in docker container or not via /.dockerenv file
+    std::ifstream dockerEnv("/.dockerenv");
```

Comment:
you can use `path_exist` to check if this path exists, which locates `be/src/fs/fs_util.h`

Reply:
@imay ok, i will use path_exist function in be/src/fs/fs_util.h file.